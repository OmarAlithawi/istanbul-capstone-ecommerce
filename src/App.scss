@import "~bootstrap/scss/bootstrap";
@import "./components/common/variables";
