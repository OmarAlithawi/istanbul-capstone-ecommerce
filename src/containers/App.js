import React from "react";
import "../App.scss";
import { RouteBlock } from "./RouteBlock";
import { Route } from "react-router-dom";
import Navbar from "../components/layout/nav/Navbar";
import Home from "../components/home/Home";
import Cart from "../components/shoppingCart/Cart";
<<<<<<< HEAD

=======
>>>>>>> shoppingCart
function App() {
  return (
    <>
      <Route path="/" component={Navbar} />
      <RouteBlock />
      <Cart />
      <Home />
<<<<<<< HEAD
      <Cart />
=======

>>>>>>> shoppingCart
    </>
  );
}

export default App;
