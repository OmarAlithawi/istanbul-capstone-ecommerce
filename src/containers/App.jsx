import React from "react";
import "../App.scss";
import HomePage from "../components/home/HomePage";
import { RouteBlock } from "./RouteBlock";

function App() {
  return (
    <>
      <HomePage />
      <RouteBlock />
    </>
  );
}

export default App;
