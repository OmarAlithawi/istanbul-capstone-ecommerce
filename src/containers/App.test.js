import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

test("renders correctly whithout crash", () => {
  const { getByText } = render(<App />);
  const text = getByText("hello");
  expect(text).toMatchSnapshot();
});
