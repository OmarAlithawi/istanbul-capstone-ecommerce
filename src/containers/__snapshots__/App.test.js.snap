// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly whithout crash 1`] = `
<h1>
  hello
</h1>
`;
